assets/dev/scss/editor/_templates-modal.scss

```diff
@@ -859,6 +859,7 @@ label.elementor-template-library-order-label {
 }
 
 #elementor-template-library-footer-banner {
+	padding-bottom: 45px;
```

Comment:
Remove the margin-bottom from `.elementor-excerpt`

Reply:
Done.